src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java

```diff
@@ -3,12 +3,17 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/** Some javadoc. */
```

Comment:
What's the purpose of these javadocs?

Reply:
To bypass whole config validation. All should have javadocs.